bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+
+//css
+import  "./css/FilterResults.css"
+
+// core components
+import ExamplesNavbar from "../components/Navbars/ExamplesNavbar.js";
+import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
+import List from "../components/List";
+import withListLoading from '../components/withListLoading';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
import withListLoading from "../components/withListLoading";
```